google/import_logging_project_exclusion_test.go

```diff
@@ -0,0 +1,30 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccLoggingProjectExclusion_importBasic(t *testing.T) {
```

Comment:
We've just started trying to get away from separating out our import tests, since it usually means we end up essentially running the exact same test twice without realizing it. Mind just moving the import check into the resource test?